modules/ROOT/pages/7.0.1-release-notes.adoc

```diff
@@ -154,12 +154,14 @@ For information on using Enhanced Skins & Icon Packs, see: xref:enhanced-skins-a
 [[bug-fixes]]
 == Bug fixes
 
-{productname} {release-version} also includes the following bug fix<es>:
+{productname} {release-version} also includes the following bug fixes:
 
-=== <TINY-vwxyz 1 changelog entry>
-// #TINY-vwxyz1
+=== Tooltips unintended shrinking and incorrectly positioned when shown in horizontally scrollable container.
+// #TINY-10797
 
-// CCFR here.
+Previously in {productname}, the toolbar tooltips within the editor were positioned absolutely. This caused tooltips to shrink and become misplaced when the editor was placed inside a container with horizontal scrolling.
```

Comment:
Should we update this to specify `absolute` rather than absolutely?

```suggestion
Previously in {productname}, toolbar tooltips within the editor were positioned `absolute`. This caused tooltips to shrink and become misplaced when the editor was placed inside a container with horizontal scrolling.
```